istio-control/istio-config/values.yaml

```diff
@@ -1,6 +1,6 @@
 galley:
   image: galley
-  replicaCount: 1
+  # replicaCount: 2
```

Comment:
Can galley safely run with multiple replicas? @costinm

Reply:
cc @ozevren

Reply:
Have you tested this?

Reply:
@ozevren: I'm running 3 replicas in production and nothing has critically failed atleast. Degredation or edge cases below my "detection threshold" might still be in there. Anything you want me to look for specifically?

Reply:
I did test galley with multiple replicas.

Reply:
But we need to confirm the stability cluster runs multiple replicas of everything - I tested manually long ago.